LINUX/CISCO_IOS_emulation/status/interface_status.xml

```diff
@@ -62,8 +62,9 @@ GigabitEthernet1 is up, line protocol is up
   <command name="IMPORT">
     <operation><![CDATA[{if isset($params.object_id)}/config/extract_part_of_simulated_file.py /config/cisco_ios/test.status "show interface {$params.object_id}" {/if}]]></operation>
     <parser>
-      <section>
-        <regexp>@^\s*(?&lt;object_id&gt;\S+) is (?&lt;interface_status&gt;up|administratively down), line protocol is (?&lt;line_protocol&gt;up|down|administratively down)@</regexp>
+      <section>GigabitEthernet3/1/2.2861 is up, line protocol is up
```

Comment:
@eyraud 
are you sure about the "GigabitEthernet3/1/2.2861 is up, line protocol is up" in <section>